src/coreComponents/linearAlgebra/interfaces/hypre/HypreInterface.cpp

```diff
@@ -18,16 +18,27 @@
 
 #include "HypreInterface.hpp"
 #include "linearAlgebra/interfaces/hypre/HyprePreconditioner.hpp"
+#include "HYPRE_utilities.h"
+#include "_hypre_utilities.h"
 
 namespace geosx
 {
 
 void HypreInterface::initialize( int & GEOSX_UNUSED_PARAM( argc ),
                                  char * * & GEOSX_UNUSED_PARAM( argv ) )
-{}
+{
+  HYPRE_Init();
+#if defined(USE_CUDA)
+  HYPRE_ExecutionPolicy default_exec_policy = HYPRE_EXEC_DEVICE;
```

Comment:
@rrsettgast this line is not needed since the handle is being set explicitly to HYPRE_EXEC_DEVICE in the subsequent line. Only need to use the default_exec_policy variable if you would be switching between exec policies at compile time.